3c.segment_cells/scripts/3.cell_segmentation.py

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This notebook focuses on trying to find a way to segment cells within organoids properly.
+# The end goals is to segment cell and extract morphology features from cellprofiler.
+# These masks must be imported into cellprofiler to extract features.
```

Comment:
These comments seem generalized to many notebooks. Consider making the comments more specific or perhaps collapsing the use of many notebooks into one or two and reusing capabilities where there is repeated code.